aipcli/token.go

```diff
@@ -27,11 +27,11 @@ type IdentityTokenFile struct {
 }
 
 func identityTokenFromConfigFile(tokenFile string) (string, error) {
-	configDir, err := os.UserConfigDir()
+	homeDir, err := os.UserHomeDir()
 	if err != nil {
 		return "", err
 	}
-	configFile := path.Join(configDir, tokenFile)
+	configFile := path.Join(homeDir, ".config", "saga", tokenFile)
```

Comment:
This is a generic and public CLI generator for AIP applications, we shouldn't add any Einride specific concepts in here

Reply:
The other issue with hardcoding the config here is that this can be used to generate several CLIs with completely different implementation details and a user has more than 1 of those in his machine, those files would be overriding each other. We need to account for this so the user can pass these values in a config so it can be CLI specific

Reply:
We have "gcloudAuthPrintIdentityToken" which binds this tool with google account auth. We didn't provide any other auth mechanism. 

So why not provide a Saga auth mechanism EXACTLY very much like the google account auth here. So if google auth is not used, then Saga auth is used. We define what Saga auth tooling conversion is (f.g. the config file location and naming patterns). If anyone is really using this tool and needs other auth methods, they can open issues and do PR requests we can provide help and support them by then.

I do not think we should try making this tool "very configurable" so to allow others to use it while slowing us down here.

It is very inconvenient that one needs to do auth login every time when switching dev/prod. I was developing a service A locally while helping debug stuff in service B in prod and I needed to constantly login backend and forth. If we just treat saga-auth the same as some gcloud auth thing then it makes lots of things easier?

Reply:
@liufuyang What you're talking about here sounds like it's best done in a private repo. This repo was created with the goal of aligning to the AIP standard without assuming anything about Einride's internal infrastructure.